src/src/CustomBuild/define_plugin_sets.h

```diff
@@ -1569,9 +1569,6 @@ To create/register a plugin, you have to :
   #ifdef USES_P076
     #undef USES_P076   // HWL8012   in POW r1
   #endif
-  #ifdef USES_P092
-    #undef USES_P092   // DL-Bus
-  #endif
   #ifdef USES_P093
```

Comment:
I guess the same issue will apply to P076, P093 and P100, missing from their TESTING sets 🤔 

I've been testing this with a wrapper `#ifndef PLUGIN_SET_TESTING` around those `#ifdef/#undef`'s, though I've not yet been able to build all variations.

Reply:
P076 & P093 are part of the energy build.

Reply:
Ok, but P100 is in TESTING set B

Reply:
P100 should get the same treatment, seems to fit now in even the (previously) most cramped builds.